pkg/pluginx/kubeconfig.go

```diff
@@ -34,3 +34,11 @@ func PersistKubeConfig(ctx context.Context, kc []byte) (string, func(context.Con
 
 	return abs, deleteFn, nil
 }
+
+// CheckKubeConfigProvided returns an error if a given kubeconfig is empty or nil.
+func CheckKubeConfigProvided(pluginName string, kubeconfig []byte) error {
```

Comment:
What do you think about renaming that to `ValidateKubeConfigProvided` (in future we can extend the validation) or `EnsureKubeConfigProvided`?